cypress/support/API/leaveAPI/leaveAPI.ts

```diff
@@ -0,0 +1,45 @@
+export default class leave {
+  static adminAddLeaveEntitlements(empNum: number) {
+    cy.request({
+      method: "POST",
+      url: "/api/v2/leave/leave-entitlements",
+      body: {
+        empNumber: empNum,
+        leaveTypeId: 7,
+        fromDate: "2023-01-01",
+        toDate: "2023-12-31",
+        entitlement: "100",
+      },
+    }).then((res) => {
+      cy.logout();
```

Comment:
why you logout inside this request? logout action not always used after the add Entitlement